src/components/scrollBar/scrollBar.api.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
I remember we wanted to avoid exposing it and keep this component internal because it's too specific for our usages